### DISCUSSION TITLE
[Q&A] 500 Can\'t connect to <host>:443 (No route to host)

### DISCUSSION THREAD

**Discussion by ricardobasilioakwel** [glpi-project/glpi-agent#736] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

My ESX servers are on version 8. I configured ESX with the GLPI Inventory Plugin using the IP address and associated the login and password. I then created a task on the GLPI server to inventory my server. It inventoried without any problems but in the task report it shows the following error:
![firefox_x8JdhCTcLi](https://github.com/user-attachments/assets/398552ac-bd82-4a33-996e-277834105759)

When I forced the inventory from the GLPI agent on the GLPI server (Debian Server 12 with the latest version of the GLPI agent 1.10), I got this error: **_500 Can\'t connect to <ServerIP>:443 (No route to host)_**. Note: It manages to collect the information from the server and update the data.

I launched the command _**glpi-agent --logger=stderr --debug --debug --force**_ and the result is as shown in the image below.
![WindowsTerminal_GL1HtcE7d1](https://github.com/user-attachments/assets/f4b3320d-0976-46e0-a118-9be9903b6c11)

As I mentioned above, the agent can get the data and updates the ESX machine in the GLPI application, but in the task status it shows the status in error.

Can you help me? I think it's a bug in the agent or plugin.

**Comment by g-bougard** [comment]
Hi @ricardobasilioakwel 

in glpi-inventory plugin log, it reports `Can't connect to` with an ip. Which ip is it ? The ESX ip ? Also the `(No route to host)` should be explicit for you: glpi-agent can't reach that ip for connectivity reason.

In the submitted debug2 log, it seems we have another message: "500 Internal Server Error: Cannot complete login due to an incorrect user name or password."

Are you sure the 2 errors are related ? To me, they seem not.
